### PR TITLE
Osc 16 latest

### PR DESCRIPTION
**Pull request by Bunhok**

### CODE REVIEW

File: app/assets/stylesheets/datatable/table.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    border-right-width: 0;
+    &.dataTable {
+      th, td {
+        &:last-child {
+          border-right-width: 1px;
+        }
+      }
+    }
+  }
+}
```
Comment (kirykr) [inline app/assets/stylesheets/datatable/table.scss:12]: @Bunhok  Can you add an empty new line here?

File: app/classes/advanced_searches/client_fields.rb [hunk 2]
```diff
@@ -13,21 +13,37 @@ def initialize(options = {})
...
+      other_group           = format_header('other')
 
       number_fields         = number_type_list.map { |item| AdvancedSearches::FilterTypes.number_options(item, format_header(item), group) }
+      number_fields         += case_history_number_type_list.map { |item| AdvancedSearches::FilterTypes.number_options(item, format_header(item), case_history_group) }
       text_fields           = text_type_list.map { |item| AdvancedSearches::FilterTypes.text_options(item, format_header(item), group) }
-      text_fields           << referee_text_fields.map { |item| AdvancedSearches::FilterTypes.text_options(item, format_header(item), referee_group) }
-      text_fields           << carer_text_fields.map { |item| AdvancedSearches::FilterTypes.text_options(item, format_header(item), carer_group) }
-      date_picker_fields    = date_type_list.map { |item| AdvancedSearches::FilterTypes.date_picker_options(item, format_header(item), group) }
-      date_picker_fields    += [['no_case_note_date', I18n.t('advanced_search.fields.no_case_note_date')]].map{ |item| AdvancedSearches::CsiFields.date_between_only_options(item[0], item[1], group) }
```
Comment (kirykr) [inline app/classes/advanced_searches/client_fields.rb:26]: @Bunhok Do we have to remove these mappings?

Comment (Bunhok) [reply app/classes/advanced_searches/client_fields.rb:26]: Yes, we do.

File: app/assets/stylesheets/datatable/table.scss (hunk 1 above)
Comment (Bunhok) [reply app/assets/stylesheets/datatable/table.scss]: Yes, I can.

**Pull request merged**